### PR TITLE
Make the experiment runnable using Krun.

### PR DESCRIPTION
**Pull request by vext01**
- Fetch and build Krun.

 - Use a script to auto-generate an importable Krun config snippet
   containing stuff that can be derived krun_ext_common.py, thus dodging
   the Python2 vs. Python3 problem.

Tested lightly with the PRE/POST commands commented out.

### CODE REVIEW

File: experiment.krun [hunk 1]
```diff
@@ -8,18 +8,23 @@
...
+
 N_EXECUTIONS = 10
 ITERATIONS_ALL_VMS = 2000
 
 MAIL_TO = []
 
-# The heap limit must be at least the value passed to to the runner in krun_ext_*.py
-HEAP_LIMIT = 1024 * 1024 * 18  # 18 GiB
-STACK_LIMIT = 1024 * 16  # 16 MiB
+# We allow 2GiB of native heap in addition the the Java heap.
```
Comment (ltratt) [inline experiment.krun]: "in addition to the Java heap"

Comment (vext01) [inline experiment.krun]: Just spotted a typo here.

**Comment by vext01** [comment]
Typo fixed.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**